README.md

```diff
@@ -2,8 +2,33 @@
 
 ## installation:
 
+
+### Linux (with anaconda)
+
+    conda install numpy protobuf libgcc
```

Comment:
`libgcc` is not required by travis. so should we remove it ? also might be better to create a `requirements.txt` file for pip no?
`pytest` is missing

Reply:
Well spotted this `libgcc` stuff ... I removed it.

Reply:
pytest is not an install requirement ..  I'll add it to setup.py as recommended by the pytest documentation